plugins/samples/redirect/tests.textpb

```diff
@@ -0,0 +1,25 @@
+# No expected path, no redirect.
+test {
+  name: "NoRedirect"
+  benchmark: false
+  request_headers {
+    input { header { key: ":path" value: "/main/somepage/otherpage" } } 
+    result { 
+      has_header { key: ":path" value: "/main/somepage/otherpage" }
+      no_header { key: "Location" }
+    }
+  }
+}
+# With expected path, expected imediate with http status code 301 and
+# redirect to other location.
+test {
```

Comment:
Can you add a test case that ensures that "/foo/" is only replaced with "/bar/" when at the beginning of the URL path?

Reply:
Done